doc/user/content/integrations/_index.md

```diff
@@ -202,23 +202,23 @@ The level of support for these tools will improve as we extend the coverage of `
+| Service            | Support level                    | Notes                                                                                                                                                                                                                                                                                               |             |
+| ------------------ | -------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- |
+| Metabase           | {{< supportLevel beta >}}        | Connect using the [PostgreSQL database driver](https://www.metabase.com/docs/latest/administration-guide/databases/postgresql.html). See the [Metabase integration page](/integrations/metabase/) for more details.                                                                                 | [](#notify) |
+| Superset           | {{< supportLevel alpha >}}       | Connect using the [PostgreSQL database driver](https://superset.apache.org/docs/databases/postgres/).                                                                                                                                                                                               | [](#notify) |
+| Preset             | {{< supportLevel alpha >}}       | Connect using the [PostgreSQL database driver](https://superset.apache.org/docs/databases/postgres/).                                                                                                                                                                                               | [](#notify) |
+| Looker             | {{< supportLevel alpha >}}       | Connect using the [PostgreSQL database driver](https://superset.apache.org/docs/databases/postgres/).                                                                                                                                                                                               | [](#notify) |
+| Google Data Studio | {{< supportLevel alpha >}}       | Connect using the [PostgreSQL database driver](https://support.google.com/datastudio/answer/7288010?hl#how-to-connect-to-postgresql&zippy=%2Cin-this-article).                                                                                                                                      | [](#notify) |
+| Tableau            | {{< supportLevel alpha >}}       | Connect using the [PostgreSQL database driver](https://help.tableau.com/current/pro/desktop/en-us/examples_postgresql.htm). Use the JDBC-based driver, which is included by default on Windows but requires an extra [driver download](https://www.tableau.com/support/drivers) on macOS and Linux. | [](#notify) |
```

Comment:
This reminds me I owe Bobby a review on #20030.

Reply:
Oh, nice, didn't realize that was up. I'll rebase that PR on this one once I merge.